### DISCUSSION TITLE
Packaging for delivery

### DISCUSSION THREAD

**Discussion by ONDC-Tech** [ONDC-Official/protocol-network-extension#40] [General]
There are different logistics delivery modes - merchant to buyer (direct), merchant to buyer (through hub). Providers support one of these modes even for hyperlocal. Packaging for the 1st mode is different from the 2nd mode and will vary for different categories.
Open questions to address:
1. Do we need to communicate the logistics delivery mode over the protocol from the logistics provider to the merchant?
2. Are there packaging standards from logistics providers for both these modes?

**Comment by ONDC-Tech** [comment]
@Brij-SA @aakasjayaraj @mahoriR @mherle @VidyaGF @NajeebMohammed https://github.com/orgs/ONDC-Official/teams/participants

**Comment by BLR-0118** [comment]
Also, shipping label can be provided by logistics provider or merchant. Do we define who provides the shipping label?
@bansalmeghaa

**Reply by bansalmeghaa** [reply]
As per the industry view and keeping in mind further categories going live, Shipping label should be provided by the merchant itself, because  in case of, lets say more than 10 shipments  to be picked up from the seller by any LP, how will the LP identify the shipments without labels.
Second, in case of off -network deliveries, how will Seller himself identify shipments,(( In multi-shipment intercity deliveries) even if delivery happens at sellers end only with no involvement of on- network LP.

How it works now  in platform based deliveries is : order come to seller-> seller packs it-> seller put the label -> info goes to LP to pick up at certain time slots.

**Comment by bansalmeghaa** [comment]
Packing standard-- there should be  min recommended guidelines/standard for packing each type of category for better user/buyer experience, for avoiding damages/ for less grievances generation and for inclusion of all sellers/merchant who are new to Ecommerce.

Packing standard/guidelines is needed because LP also ask to pack different category items in a certain manner to avoid damages in the transit.
